core/src/main/resources/AppConstants.properties

```diff
@@ -23,7 +23,6 @@ instance.version=
 #instance.build_id=not-used
 
 configurations.names.application=IAF_Util,IAF_WebControl,${configurations.names}
-configurations.names.application=IAF_WebControl,${configurations.names},IAF_Util
```

Comment:
De laatste property die gezet is wordt gebruikt, je hebt de verkeerde verwijderd..